docs/pages/setup/guides/fluentd.mdx

```diff
@@ -321,6 +353,29 @@ client_crt = "auth.crt"
 root_cas = "auth.cas"
 ```
 
+</TabItem>
+<TabItem scope={["oss", "enterprise"]} label="Helm chart">
+
+Use the following template to create values file:
+
+```yaml
+eventHandler:
+  storagePath: "./storage"
+  timeout: "10s"
+  batch: 20
+  namespace: "default"
+
+fluentd:
+  url: "https://localhost:8888/test.log"
+  caPath: "/home/sasha/scripts/event-handler/ca.crt"
```

Comment:
Are these values correct? Should it be the path inside the secret or does this secret get mounted?